functions/go/Makefile

```diff
@@ -4,7 +4,7 @@
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
-#      http://www.apache.org/licenses/LICENSE-2.0
+#\t\t\thttp://www.apache.org/licenses/LICENSE-2.0
```

Comment:
some extra tabs (probably from your IDE)

Reply:
Fixed, thanks for the catch!